From fe1754cc6ed3b582c3329a4afb691a4e3d195d4b Mon Sep 17 00:00:00 2001
From: Jon Nettleton <jon@solid-run.com>
Date: Tue, 31 Jan 2023 09:42:39 +0100
Subject: [PATCH 8/8] net: ethernet: fec: fix missing header include

Fix compilation failure on pinctrl_pm_select_default_state
due to missing header.

Signed-off-by: Jon Nettleton <jon@solid-run.com>
---
 drivers/net/ethernet/freescale/fec_uio.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/net/ethernet/freescale/fec_uio.c b/drivers/net/ethernet/freescale/fec_uio.c
index aa558a784f73..db53b3bb3106 100644
--- a/drivers/net/ethernet/freescale/fec_uio.c
+++ b/drivers/net/ethernet/freescale/fec_uio.c
@@ -6,6 +6,7 @@
 #include <linux/kernel.h>
 #include <linux/of_platform.h>
 #include <linux/uio_driver.h>
+#include <linux/pinctrl/consumer.h>
 #include <linux/pm_runtime.h>
 #include <linux/errno.h>
 #include <linux/netdevice.h>
-- 
2.39.0

